pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "obc_rate_noise_plots"
+dynamic = ["version"]
+dependencies = []
+
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project.urls]
+Homepage = "https://github.com/sot/obc_rate_noise_plots"
+
+[project.scripts]
+obc-rate-noise-plots = "obc_rate_noise_plots.plot_recent_rates:main"
```

Comment:
I don't have experience yet with pyproject.toml files, but I'm assuming here that this is fully replacing setup.py for this project (since there's no setup.py).  For projects that move to toml is there anything else I should look at?  I'm throwing that out to @taldcroft .